app/__init__.py

```diff
@@ -77,9 +89,12 @@ def create_app():  # noqa: C901  pylint: disable=too-complex
     )
 
     application.eq['database'] = Database(
-        application.config['EQ_SERVER_SIDE_STORAGE_DATABASE_URL'],
-        application.config['EQ_SERVER_SIDE_STORAGE_DATABASE_SETUP_RETRY_COUNT'],
-        application.config['EQ_SERVER_SIDE_STORAGE_DATABASE_SETUP_RETRY_DELAY_SECONDS'],
+        database_driver=application.config['EQ_SERVER_SIDE_STORAGE_DATABASE_DRIVER'],
+        database_host=application.config['EQ_SERVER_SIDE_STORAGE_DATABASE_HOST'],
+        database_port=application.config['EQ_SERVER_SIDE_STORAGE_DATABASE_PORT'],
+        database_name=application.config['EQ_SERVER_SIDE_STORAGE_DATABASE_NAME'],
```

Comment:
I think we should drop the `database_` from the driver, host and port params. `database_name` should stay though as it's less clear what that is if we drop the `database_` from it.